release.sh

```diff
@@ -17,20 +17,15 @@ if [ -z "${VERSION}" ]; then
   exit 1
 fi
 
+STABLE_TAGGED="$(echo $VERSION | grep -o '-stable')"
+
 if [ -z "${CHANNEL}" ]; then
   echo "Channel not set, will release as unstable"
-  echo "To promote to a new channel, use the equinox release tool:"
-  echo '$ equinox publish --token $TOKEN --app $APP  --channel stable --release 0.0.39'
   CHANNEL="unstable"
 fi
 
-if [ "${CHANNEL}" == "preview" ]; then
-  echo "Channel is preview, setting version to timestamp"
-  VERSION="$(date +"%Y%m%d%H%M%S")"
-elif [ "${CHANNEL}" == "stable" ]; then
-  echo "To promote to stable, use the equinox release tool:"
-  echo '$ equinox publish --token $TOKEN --app $APP  --channel stable --release $VERSION'
-  exit 0
+if [ "${STABLE_TAGGED}" == "-stable" ]; then
+    CHANNEL="stable"
```

Comment:
We should probably strip `-stable` from the version string

Reply:
Sure